test/index.js

```diff
@@ -276,7 +276,34 @@ describe('pbnormalizr', () => {
                 .should.eql(address);
         });
 
-        it('will return null if the protobuf doesn\'t have all the required fields', () => {
+        it('will return null if the protobuf doesn\'t pass validation', () => {
+            // Test with validator that checks for required fields
+            let entityHasValueForField = (entity, field) => {
```

Comment:
I think it would be useful if the library exported some of these default validators like "requiredFields". I assume we'll need this validators on our end.